### PR TITLE
Desktop_version_one

### PR DESCRIPTION
**Pull request by Kal2403**
1. Added CSS media query for Desktop
2. Added CSS Flexbox and Grid
3. Added in HTML, Technologys to section-p3

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi @Kal2403 👋👋👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some linter test failing🥲. Kindly fix all your linter errors and resubmit your project👍.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -414,3 +415,209 @@
...
+    width: 100%;
+  }
+
+  .card {
+    width: 1156px;
+    height: 496px;
+    display: flex;
+    flex-direction: row;
+    border: 1px solid rgba(223, 225, 230, 1);
+  }
```
Comment (LionRouge1) [inline style.css:525]: - [ ] Kindly note the color and the gap between the project title, information, details and technologies in the Figma design and the gap between the project image and description.
- [ ] Make sure the project description content is aligned to the left 
- [ ] Please stick to the Figma design as much as possible as requested in the project requirement.

### Your design :point_down: 
Screenshot from 2023-05-04 10-07-11
### Figma design :point_down: 
Screenshot from 2023-04-06 10-25-14

File: style.css [hunk 2]
```diff
@@ -414,3 +415,209 @@
...
+
+  .frame {
+    justify-content: space-evenly;
+  }
+
+  .section-p3 {
+    flex-direction: row;
+    height: 668px;
+    align-items: center;
+  }
```
Comment (LionRouge1) [inline style.css:561]: - [ ] Nice work implementing the "About Myself" section :clap: However make the left content have 40% and the right content 60% of the page's width to follow the Figma design.
### Your design :point_down: 
Screenshot from 2023-05-04 10-21-51
### Figma design :point_down: 
Screenshot from 2023-04-05 06-38-27

File: style.css [hunk 3]
```diff
@@ -414,3 +415,209 @@
...
+  .dropdown-content li {
+    border: none;
+    padding: 5px;
+    background: #f7f7f9;
+    margin-bottom: 30px;
+  }
+
+  .dropdown-content li img {
+    padding-right: 8px;
+  }
```
Comment (LionRouge1) [inline style.css:595]: - [ ] Kindly display each element in the dropdown in a column and remove the bottom line to follow the Figma design as requested in the project requirement.
### Your design :point_down: 
Screenshot from 2023-05-04 11-17-39
### What it is supposed to look like:point_down:
Screenshot from 2023-05-04 11-18-39

File: style.css [hunk 4]
```diff
@@ -104,7 +104,7 @@
   }
 
   main {
-    background-color: #e5e5e5;
+    background-color: #fff6f6b7;
```
Comment (LionRouge1) [inline style.css:107]: - [ ] Your background color is different from the one on the Figma design(`#f4f5f7`). Kindly check it to follow the Figma design.

File: style.css [hunk 5]
```diff
@@ -104,7 +104,7 @@
...
   }
 
   .section-p1 {
```
Comment (LionRouge1) [inline style.css:110]: - [ ] Kindly reduce the space between the "headline" section and the "Works" section it is a little bigger compared to the Figma design.
### Your design :point_down: 
Screenshot from 2023-05-04 11-27-51
### What it is supposed to look like :point_down: 
Screenshot from 2023-05-04 11-37-35

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Kal2403, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job using semantics tags :100: 
- Nice work using media query for the desktop version :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [ ] Kindly use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels to make your portfolio responsive.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] You can also consider making your commit messages more professional. A commit message should be written in imperative tense and capitalized as shown below:
226624434-f307e11a-3a3a-4b72-bf72-95c7ac702293

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**